lib/converters.js

```diff
@@ -25,6 +25,22 @@ function indent(content, spaces, ignoreFirst) {
     }).join("\n");
 }
 
+function emphasisFormats( text ) {
+  return {
+    strong: '**',
```

Comment:
Please use 4 space indent and [double quotes](https://github.com/IonicaBizau/code-style#quotes-speech_balloon). Also put the commas [at the beginning of the lines](https://github.com/IonicaBizau/code-style#commas).